build-scripts/esp-idf/wamr/CMakeLists.txt

```diff
@@ -2,10 +2,10 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 # Set WAMR's build options
-if("${IDF_TARGET}" STREQUAL "esp32c3")
-    set(WAMR_BUILD_TARGET "RISCV32")
-else()
+if("${IDF_TARGET}" STREQUAL "esp32")
```

Comment:
Using STREQUAL is invalid: STREQUAL checks whether the input `==esp32`, but not "==esp32c3 or ==esp32c6". And it sets the build target to xtensa then, but not riscv32.
How about:
```cmake
if ("${IDF_TARGET}" STREQUAL "esp32c3" OR "${IDF_TARGET}" STREQUAL "esp32c6")
    set(WAMR_BUILD_TARGET "RISCV32")
else ()
    set(WAMR_BUILD_TARGET "XTENSA")
endif ()
```

Reply:
yeah, I forgot ESP32S3 is also XTENSA structure. I will fix it now. Thx. :)